treeherder/model/derived/jobs.py

```diff
@@ -1338,6 +1339,22 @@ def load_job_data(self, data, raise_errors=False):
 
         retry_job_guids = []
 
+        # get the tier-2 data signatures for this project.
+        # if there are none, then just return an empty list
+        tier_2_signatures = []
+        try:
+            tier_2 = ExclusionProfile.objects.get(name="Tier-2")
+            # tier_2_blob = json.loads(tier_2['flat_exclusion'])
+            tier_2_signatures = tier_2.flat_exclusion[self.project]
```

Comment:
An optimization here could be to use a set for the tier2 signatures, so that the IN check performed for every job will be quicker